src/test/javascript/spec/component/admin/user-management.component.spec.ts

```diff
@@ -44,7 +77,7 @@ describe('UserManagementComponent', () => {
     // The admin module is lazy loaded - we therefore need a dummy test to load
     // the module and verify that there are no dependency related issues.
     it('should render a component from the admin module', () => {
-        expect(comp).toBeDefined();
+        expect(comp).not.toBe(null);
     });
 
     it('should parse the user search result into the correct component state', inject(
```

Comment:
I've never seen the fakeAsync written down like this. Does it also work if written down normally without the inject?

Reply:
yes, works. I'll change it